pkgs/applications/version-management/gitlab/gitlab-shell/deps.nix

```diff
@@ -68,10 +113,37 @@
     fetch = {
       type = "git";
       url = "https://github.com/getsentry/raven-go";
-      rev = "v0.1.0";
+      rev = "v0.1.2";
       sha256 = "1dl80kar4lzdcfl3w6jssi1ld6bv0rmx6sp6bz6rzysfr9ilm02z";
```

Comment:
This doesn't look right, `sha256` should have been updated as well.

Reply:
Indeed, that's weird. I guess it worked for me since the old package was already in my store. I know I ran `nix-collect-garbage`, but I might have had a `result` lying around somewhere...

Reply:
Ah, nope, this is correct: https://github.com/getsentry/raven-go/releases

No idea why it's tagged like that, but it is :confused:

Reply:
Ok, thanks for checking.